spec/util/enforce_noecho_parameter_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+require 'cfn-model'
+require 'cfn-nag/util/enforce_noecho_parameter'
+
+describe 'no_echo_parameter_without_default', :rule do
+
+  context 'test function no_echo_parameter_without_default' do
+    it 'returns false if there is a problem' do
+      cfn_model = CfnParser.new.parse read_test_template('json/rds_instance/rds_instance_no_echo_with_default_username.json')
```

Comment:
totally cool to take the "sociable" unit testing approach here.... but Q: is there 100% coverage on the method (2 specs cover the 4 or so outcomes?)  if not, you can probably mock the model to get the last nook/cranny conditional...

Reply:
80% coverage, but I can get it to 100. Will do